integration-cli/docker_cli_export_import_test.go

```diff
@@ -37,12 +31,9 @@ func (s *DockerSuite) TestExportContainerAndImportImage(c *check.C) {
 	}
 
 	cleanedImageID := strings.TrimSpace(out)
-
-	inspectCmd = exec.Command(dockerBinary, "inspect", cleanedImageID)
-	if out, _, err = runCommandWithOutput(inspectCmd); err != nil {
-		c.Fatalf("output should've been an image id: %s, %v", out, err)
+	if cleanedImageID != "" {
```

Comment:
shouldn't this be `==` instead of `!=`  ? I would think empty output would be wrong.